### ISSUE TITLE
Support standard F5 keyboard shortcut for 'refresh' action

### ISSUE THREAD

**Issue by johndowns** [microsoft/AzureStorageExplorer#2920]
**Is your feature request related to a problem? Please describe.**
This is a usability issue. When I have a queue, blob container, or table open, I frequently need to refresh the view to see the latest information. The standard convention is to allow for the F5 keyboard shortcut to do this, but this does not appear to be bound to the 'refresh' action. Instead, I have to use the mouse to click on the 'Refresh' icon on the toolbar. If I am heavily keyboard-focused (which is often the case when I am writing code and switching windows to view the results in Storage Explorer) this is quite frustrating.

**Describe the solution you'd like**
Please create a new keyboard shortcut to bind the F5 key to the 'refresh' action.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by craxal** [comment]
@johndowns Refresh should be keyboard-accessible by pressing Ctrl+R. Does that work?

**Comment by johndowns** [comment]
@craxal yes it does - but I still think that this should be bound to F5, since this is the standard Windows keyboard shortcut for refreshing ([source](https://support.microsoft.com/en-nz/help/12445)).

**Comment by ImrePyhvel** [comment]
Not sure it is common for F5 to refresh everywhere but I do agree that it is very common for query editors to run the current (or currently selected part) of query. Currently CosmosDB Query tab does not have F5 bound to "Execute query" button and Ctrl-R does not work either. 

Imho F5 should work at least in CosmosDB Query tab.

**Comment by MRayermannMSFT** [comment]
@johndowns, for the record, the docs you linked to also show ctrl+r as a standard refresh keyboard shortcut:

<img width="687" alt="image" src="https://user-images.githubusercontent.com/21112697/84065106-9e95ab80-a978-11ea-9683-34a007ea8ac7.png">

We're going to put this in future for now to gauge interest from other users as well.

**Comment by thomaseyde** [comment]
Also, please move the refresh button to the left of the toolbar. It is more used than "Folder Statistics", "Manage History" and "Undelete". Even better, allow us to customise the toolbar.

**Comment by craxal** [comment]
@thomaseyde Being able to customize the toolbar sounds like a great idea.

**Comment by stonstad** [comment]
Why not support both?  Nearly all browsers support Control + F5 and Control + R.  Honestly, I was surprised to learn that Control + R is even common. The pinky finger dexterity required for that shortcut is clearly inferior to mashing F5.   :)
